base/get_database.py

```diff
@@ -133,3 +133,8 @@ def get_usercache_db():
     current_db = MongoClient().Stage_database
     UserCache = current_db.Stage_usercache
     return UserCache
+
+def get_fake_trips_db():

```

Comment:
I think that it is fine for the fake trips to be separated. If you do it this way, though, scripts written to work on real data won't work on fake data.

Let me give you a concrete use case. The recommendation pipeline tests ensure that the recommendation pipeline reads sections and processes them correctly. But the recommendation pipeline reads sections from the section DB.

So if you store fake trips in the fakeDB and then run the recommendation pipeline, it won't do anything.

There are a couple of other ways of doing this:

1. cheap and easy: use a unique uuid prefix or suffix to indicate that the records belong to a fake person.
1. more complicated but more correct: create a factory method that returns different values of get_db() based on the context/config. Then, during normal operation, have it return Staging_database and during test operation, have it return Test_database or Fake_database. This way, we can run tests on the same database as our real data without any conflicts.

But in both cases, you will be writing into the trip_db and section_db collections.